### PR TITLE
Add target=“_parent” to download links

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: src/themes/elife/lib/downloads.ts [hunk 1]
```diff
@@ -60,12 +64,20 @@ const buildMenu = (
         create(
           'li',
           null,
-          create('a', { href: `${deriveUrl('bibtex', articleId)}` }, 'BibTeX')
+          create(
+            'a',
+            { href: `${deriveUrl('bibtex', articleId)}`, target: '_parent' },
```
Comment (discodavey) [inline src/themes/elife/lib/downloads.ts]: This just downloads a .bib file. Does this need a parent target?

File: src/themes/elife/lib/downloads.ts [hunk 2]
```diff
@@ -60,12 +64,20 @@ const buildMenu = (
...
+            'BibTeX'
+          )
         ),
         create(
           'li',
           null,
-          create('a', { href: `${deriveUrl('ris', articleId)}` }, 'RIS')
+          create(
+            'a',
+            { href: `${deriveUrl('ris', articleId)}`, target: '_parent' },
```
Comment (discodavey) [inline src/themes/elife/lib/downloads.ts]: Same as Bibtex. Does this need a parent target on it?

File: src/themes/elife/lib/downloads.ts [hunk 3]
```diff
@@ -95,7 +107,10 @@ const buildMenu = (
           null,
           create(
             'a',
-            { href: `${deriveUrl('papers', articleId, articleTitle)}` },
+            {
+              href: `${deriveUrl('papers', articleId, articleTitle)}`,
+              target: '_parent',
```
Comment (discodavey) [inline src/themes/elife/lib/downloads.ts]: Does not seem to be doing anything. How can I test this one?

File: src/themes/elife/lib/downloads.ts [hunk 4]
```diff
@@ -40,45 +42,17 @@ const buildMenu = (
...
-            'a',
-            {
-              href: pdfUrl,
-              target: '_parent',
-              'data-is-download-pdf-link': true,
-            },
-            'Article PDF'
-          )
-        )
-      ),
```
Comment (discodavey) [inline src/themes/elife/lib/downloads.ts:55]: This has now returned the list item within the list item rather than them being separate

Comment (nlisgo) [reply src/themes/elife/lib/downloads.ts:55]: This is addressed in this PR: https://github.com/stencila/thema/pull/200

Comment (nlisgo) [reply src/themes/elife/lib/downloads.ts:55]: Ignore me. will resolve this

**Comment by nlisgo** [comment]
> change the merge commit message to something like `feat(eLife): Add target=“_parent” to download links`
> I'll leave this PR for you to merge once you're ready.

@alex-ketch, I've resolved the merge conflict. I will ensure the commit merge message is `feat(eLife): Add target=“_parent” to download links`

**Pull request merged**